core/storage/statistics/gae_models.py

```diff
@@ -972,29 +972,66 @@ class ExplorationIssuesModel(base_models.BaseModel):
     """Model for storing the list of playthroughs for an exploration grouped by
     issues.
     """
+    # ID of exploration.
+    exp_id = ndb.StringProperty(indexed=True)
```

Comment:
Why do these properties not have "required=True"?

Reply:
@seanlip It got missed. I'll add It in my other PR.

Reply:
OK. Please file an issue to do this so we don't lose track, and link to the other PR in this comment so that there's a reference trail.